src/System.Collections.Concurrent/tests/System.Collections.Concurrent.Tests.csproj

```diff
@@ -98,6 +98,8 @@
     <Compile Include="$(CommonTestPath)\System\Collections\IEnumerable.Generic.Serialization.Tests.cs">
       <Link>Common\System\Collections\IEnumerable.Generic.Serialization.Tests.cs</Link>
     </Compile>
+    <Compile Include="ConcurrentBagTests.netstandard1.7.cs" />
```

Comment:
I just filed issue https://github.com/dotnet/corefx/issues/15298 to fix netstandard test build configuration. Thanks for indirectly reminding and point it out to me.